server/src/test/java/io/unitycatalog/server/base/table/BaseTableCRUDTest.java

```diff
@@ -69,11 +69,32 @@ public void testTableCRUD() throws IOException, ApiException {
     assertEquals(1, columnInfos2.stream().filter(c -> c.getName().equals("as_int")).count());
     assertEquals(1, columnInfos2.stream().filter(c -> c.getName().equals("as_string")).count());
 
-    // List tables
-    System.out.println("Testing list tables..");
-    Iterable<TableInfo> tableInfos =
+    // Create multiple tables
+    List<TableInfo> createdTables = createMultipleTestingTables(111);
+
+    // List tables with pagination - default is 100 tables per page
+    System.out.println("Testing list tables with pagination..");
+    Iterable<TableInfo> tableInfosWithPagination =
         tableOperations.listTables(TestUtils.CATALOG_NAME, TestUtils.SCHEMA_NAME);
-    assertTrue(TestUtils.contains(tableInfos, tableInfo2, table -> table.equals(tableInfo2)));
+    assertEquals(100, TestUtils.getSize(tableInfosWithPagination));
+
+    // List tables with result sorted by name
+    System.out.println("Testing list tables sorted by name");
+    Iterable<TableInfo> tableInfosSortedByName =
+        tableOperations.listTables(TestUtils.CATALOG_NAME, TestUtils.SCHEMA_NAME);
+    List<TableInfo> sortedTableList = new ArrayList<>();
+    tableInfosSortedByName.forEach(sortedTableList::add);
+    for (int i = 1; i < sortedTableList.size(); i++) {
+      assertTrue(
+          sortedTableList.get(i - 1).getName().compareTo(sortedTableList.get(i).getName()) <= 0);
+    }
```

Comment:
can we also test getting the next page and check if there are 11 expected entries ?

Reply:
This test is added to SdkTableCRUDTest. 
Pagination doesn't seem to be implemented in cli.